full-service/src/service/transaction.rs

```diff
@@ -468,6 +468,22 @@ where
         let conn = pooled_conn.deref_mut();
 
         exclusive_transaction(conn, |conn| {
+            if Account::get(&AccountID(account_id_hex.to_string()), conn)?
+                .require_spend_subaddresses
+            {
+                if spend_from_subaddress.is_none() {
+                    return Err(TransactionServiceError::TransactionBuilder(WalletTransactionBuilderError::InvalidArgument(
```

Comment:
❓ Does `InvalidArgument()` make sense for a missing argument?  Kind of depends how one looks at it, as it's "invalid json request" so perhaps it's sufficient.

I see the following, but it doesn't appear to be used in the code base
```rust
    /// Attempting to build a transaction from a TXO without a subaddress: {0}
    NullSubaddress(String),
```

Reply:
Oh wow good find

Reply:
I'm stealing this error for our purposes. From context, I imagine maybe this was plumbed originally so you couldn't spend an orphaned TXO, but since it's not used anywhere, we'll commandeer it.